OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs

```diff
@@ -779,9 +625,43 @@ public MappingProfile()
             .ForMember(dest => dest.SettlementsIds, opt => opt.Ignore());
     }
 
-    private IMappingExpression<TSource, TDestination> CreateSoftDeletedMap<TSource, TDestination>()
+    public IMappingExpression<TSource, TDestination> CreateSoftDeletedMap<TSource, TDestination>()
         where TSource : class
         where TDestination : class, ISoftDeleted
         => CreateMap<TSource, TDestination>()
             .ForMember(dest => dest.IsDeleted, opt => opt.Ignore());
-}
\ No newline at end of file
+
+    private IMappingExpression<Provider, T> AddCommonProvider2ProviderBaseDto<T>(IMappingExpression<Provider, T> mappings)
+        where T : ProviderBaseDto
+        => mappings
+            .ForMember(dest => dest.ActualAddress, opt => opt.MapFrom(src => src.ActualAddress))
+            .ForMember(dest => dest.LegalAddress, opt => opt.MapFrom(src => src.LegalAddress))
+            .ForMember(dest => dest.Institution, opt => opt.MapFrom(src => src.Institution))
+            .Apply(IgnoreAllImages)
+            .ForMember(dest => dest.ImageIds, opt => opt.MapFrom(src => src.Images.Select(x => x.ExternalStorageId)))
```

Comment:
Can we use here ` MapImageIds` method?

Reply:
Thanks for pointing this, as a result one more issue fixed (related to initially getter-only destination property described by interface)